docker/docker-compose-aspnetcore/redis/oats.redis-lpush.yaml

```
@@ -0,0 +1,16 @@
+docker-compose:
+  generator: lgtm
+  files:
+    - ../docker-compose.oats.yml
+input:
+  - path: /api/Redis/LeftPush
+interval: 5000ms
```

```suggestion
include:
  - ./oats-template.yml
input:
  - path: /api/Redis/LeftPush
```

Maybe we should also use the same approach here as for HTTP tests, utilizing the configuration matrix?